ext/mysqli/mysqli.stub.php

```diff
@@ -1457,7 +1457,7 @@ function mysqli_fetch_column(mysqli_result $result, int $column = 0): null|int|f
 
 function mysqli_field_count(mysqli $mysql): int {}
 
-function mysqli_field_seek(mysqli_result $result, int $index): bool {}
+function mysqli_field_seek(mysqli_result $result, int $index): true {} // TODO make return type void
```

Comment:
its alias should also be adjusted, otherwise `./build/gen_stub.php --verify` will emit an error:
```
mysqli_result::field_seek() and mysqli_field_seek() must have the same return type
```

Reply:
And how do I fix it without adding `true` return type to the method? Would `@return true` work?

Reply:
I changed the return type to true. Maybe we should change it straight away to void?

Reply:
The issue is ``true`` to ``void`` is a BC break as a ``void`` function returns ``null`` so if the return value of the function is used and compared against, a different code path will then be taken.